### PR TITLE
Bugfix FE 329 new language schema

### PR DESCRIPTION
**Pull request by frankyma**
Updated the queries and components in response to the new `Language` schema, just did a 1 to 1 mapping, no new UI implemented

**Testing**
Language List Item and Language Engagement List Item scenes should work as expected

### CODE REVIEW

File: src/components/LanguageEngagementListItemCard/LanguageEngagementListItem.graphql [hunk 1]
```diff
@@ -9,13 +9,15 @@ fragment LanguageEngagementListItem on LanguageEngagement {
       displayName {
         value
       }
-      ethnologuePopulation {
-        value
+      ethnologue {
+        population {
+          value
+        }
```
Comment (CarsonF) [inline src/components/LanguageEngagementListItemCard/LanguageEngagementListItem.graphql:15]: Not necessary at this view. `population` is the calculated value of either `populationOverride` or `ethnologue.pupulation`.

File: src/components/LanguageEngagementListItemCard/LanguageEngagementListItemCard.tsx [hunk 2]
```diff
@@ -79,9 +79,8 @@ export const LanguageEngagementListItemCard: FC<LanguageEngagementListItemCardPr
   const language = props.language.value;
   const name = language?.name.value ?? language?.displayName?.value;
   const population =
-    language?.organizationPopulation?.value ??
-    language?.ethnologuePopulation?.value;
-  const rodNumber = language?.rodNumber?.value;
+    language?.population.value ?? language?.ethnologue.population.value;
```
Comment (CarsonF) [inline src/components/LanguageEngagementListItemCard/LanguageEngagementListItemCard.tsx:82]: Not necessary, see other comment.

File: src/components/LanguageEngagementListItemCard/LanguageEngagementListItemCard.tsx [hunk 3]
```diff
@@ -79,9 +79,8 @@ export const LanguageEngagementListItemCard: FC<LanguageEngagementListItemCardPr
...
+  const rodNumber = language?.registryOfDialectsCode.value;
```
Comment (CarsonF) [inline src/components/LanguageEngagementListItemCard/LanguageEngagementListItemCard.tsx:83]: I'd rather not leave "rodNumber" hanging around. All of those instances should be replaced.

File: src/components/LanguageListItemCard/LanguageListItemCard.tsx [hunk 4]
```diff
@@ -67,14 +66,14 @@ export const LanguageListItemCard: FC<LanguageListItemCardProps> = ({
             <DisplaySimpleProperty
               LabelProps={{ color: 'textSecondary' }}
               label="Ethnologue"
-              value={language?.ethnologueName?.value}
+              value={language?.ethnologue.name.value}
```
Comment (CarsonF) [inline src/components/LanguageListItemCard/LanguageListItemCard.tsx:69]: I believe we are actually looking for `ethnologue.code` here which is a 3 digit ISO 639-3 code.
Can you add that to the label too?

File: src/components/LanguageEngagementListItemCard/LanguageEngagementListItem.graphql (hunk 1 above)
Comment (frankyma) [reply src/components/LanguageEngagementListItemCard/LanguageEngagementListItem.graphql]: Changed

File: src/components/LanguageEngagementListItemCard/LanguageEngagementListItemCard.tsx (hunk 2 above)
Comment (frankyma) [reply src/components/LanguageEngagementListItemCard/LanguageEngagementListItemCard.tsx]: removed

File: src/components/LanguageListItemCard/LanguageListItemCard.tsx (hunk 4 above)
Comment (frankyma) [reply src/components/LanguageListItemCard/LanguageListItemCard.tsx]: Changed

File: src/components/LanguageEngagementListItemCard/LanguageEngagementListItemCard.tsx (hunk 3 above)
Comment (frankyma) [reply src/components/LanguageEngagementListItemCard/LanguageEngagementListItemCard.tsx]: Removed

**Comment by frankyma** [comment]
@CarsonF ready

**Pull request merged**